app/models/manageiq/providers/kubevirt/infra_manager/kubernetes_event_monitor.rb

```diff
@@ -0,0 +1,6 @@
+class ManageIQ::Providers::Kubevirt::InfraManager::KubernetesEventMonitor < ManageIQ::Providers::Kubernetes::ContainerManager::KubernetesEventMonitor
```

Comment:
I wonder do we need this class at all?  It seems like we could just do the connection in the Runner

Reply:
We do need the other methods provided by the class https://github.com/ManageIQ/manageiq-providers-kubernetes/blob/master/app/models/manageiq/providers/kubernetes/container_manager/kubernetes_event_monitor.rb for the event catcher. Only reason we need to override `inventory` is because kubevirt needs to call `parent_manager` to access the k8s connection.

An alternative would be to use kube client directly here https://github.com/ManageIQ/manageiq-providers-kubernetes/blob/master/app/models/manageiq/providers/kubernetes/container_manager/kubernetes_event_monitor.rb#L10 instead of going through the kubernetes container manager. Thoughts?

Reply:
Ah okay :+1: